core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -529,15 +518,14 @@ class KafkaApisTest {
     new IncrementalAlterConfigsRequest.Builder(resourceMap, false)
   }
 
+  // FIXME: Unused arg
```

Comment:
Does this still have to be addressed? Either way, the FIXME needs to be removed.